specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/examples/ApplicationGroup/ApplicationGroupCreate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "namespaceName": "sayan-ua-test-eh-system-1",
+    "resourceGroupName": "sayantest",
+    "api-version": "2022-01-01",
+    "subscriptionId": "e8baea74-64ce-459b-bee3-5aa4c47b3ae3",
+    "applicationGroupName": "appGroup1",
+    "parameters": {
+      "properties": {
+        "appGroupIdentifier": "SASKey=RootManageSharedKey",
```

Comment:
We should add 1 more appGroup to cover AADAppKey, which is "AADAppID="

Reply:
Since this is create payload example, we can only create 1 app group at a time. Hence, adding this example in get all call.